### PR TITLE
Translate updates/dialogs-policy.md into Chinese, resolve #4383

### LINKED ISSUE #4383: Translate updates/dialogs-policy.md into Chinese

### PR DESCRIPTION
**Pull request by wayou**
this PR also includes another translation wich is in another pending PR #4331

### CODE REVIEW

File: src/content/zh-cn/updates/2017/03/dialogs-policy.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Synchronous, app-modal JavaScript dialogs are commonly (and unfortunately) used to harm users. Because of this, the Chromium team highly recommends that you not use JavaScript dialogs.
+
+{# wf_updated_on: 2017-03-24 #}
+{# wf_published_on: 2017-03-24 #}
+{# wf_tags: policy,dialog,javascript #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: Synchronous, app-modal JavaScript dialogs are commonly (and unfortunately) used to harm users. Because of this, the Chromium team highly recommends that you not use JavaScript dialogs. #}
+
```
Comment (jpmedley) [inline src/content/zh-cn/updates/2017/03/dialogs-policy.md]: Thanks for your help. Can you please translate the description and featured_snippet?

**Comment by jpmedley** [comment]
Is this ready to be merged?

**Comment by Tess98** [comment]
http://ideone.com/8Zy53f
please guide who did this. i am the owner of the account gmail and this  confidential email is public . where the assets of gmail are and the esecurity . i have full right to know, rest you can have a check on gmail account of mine .

**Comment by Tess98** [comment]
i would request you to please make me understand how who location was this script generated from my email account through hacking or whatsoever . i seriously have started hating chinese. no moral , no lawfulful, no contractual grounds ,

**Pull request merged**